### ISSUE TITLE
[TF2] [Feature Request] Change the equip region of the 'Sharp Chest Pain' to the one used by grenade cosmetics

### ISSUE THREAD

**Issue by Kibblekip** [ValveSoftware/Source-1-Games#6371]
Currently, the [Sharp Chest Pain](https://wiki.teamfortress.com/wiki/Sharp_Chest_Pain) uses the equip region taken by coats (i.e. [Cold Snap Coat](https://wiki.teamfortress.com/wiki/Cold_Snap_Coat), [Veteran's Attire](https://wiki.teamfortress.com/wiki/Veteran%27s_Attire), [Pathfinder](https://wiki.teamfortress.com/wiki/Pathfinder)) despite not visually clipping with the majority of these cosmetics.

In contrast, despite visually replacing the grenades on the Soldier's chest strap and clipping heavily, it does not conflict with most cosmetics that use the grenades equip region (i.e. [Ornament Armament](https://wiki.teamfortress.com/wiki/Ornament_Armament), [Handy Canes](https://wiki.teamfortress.com/wiki/Handy_Canes)).

I feel it would make sense to adjust the equip region to the one used by the aforementioned grenades rather than the one used by coats, due to the aforementioned visual clipping, as well as simply allowing for more customization, on top of being a relatively easy fix in general. Thank you for reading.

**Comment by DiskIntegrity** [comment]
TL;DR open `items_game.txt` and edit line `89534` from `"soldier_coat"` to `"grenades"`

**Comment by DRAG4YOU** [comment]
Yes please, I want to blow up with a knife.
